k6/index.html

```diff
@@ -56,6 +56,10 @@ <h2>知っているクリニックを探す</h2>
             </form>
           </div>
 
+          <div id="map-search" class="contents">
```

Comment:
```suggestion
          <div id="map-search">
```

すみません、構造変えたので contents クラスを外してもらえればOKなはずです

Reply:
@ktakayama ご確認ありがとうございます。こちらの修正で手元でもズレが治りました！
他に直した方がいい点ありますか?

Reply:
ありがとうございます〜〜
どこかで体裁整えたくなるかもですが、いったん良いのではないでしょうか

maps.htmlのページが、iframeしか表示してないので、indexから直接マイマップのページに飛ばしちゃうっていうのもありなのかもなとは思いました

Reply:
> iframeしか表示してないので、indexから直接マイマップのページに飛ばしちゃう

百理ありますね 👀 修正しときます。